sdcm/cluster_aws.py

```diff
@@ -329,7 +330,16 @@ def restart(self):
         # When using storage optimized instances like i2 or i3, the data on disk is deleted upon STOP. therefore, we
         # need to setup the instance and treat it as a new instance.
         if any(ss in self._instance.instance_type for ss in ['i3', 'i2']):
-            self.remoter.run('sudo rm -f /etc/scylla/ami_configured')
+            clean_script = dedent("""
+                sudo rm -f /etc/scylla/ami_configured
+                sudo sed -e '/.*scylla/ s/^/# /g' -i /etc/fstab
+                sudo sed -e 's/replace_address_on_first_boot/#replace_address_on_first_boot/g' -i /etc/scylla/scylla.yaml
+            """)
+            self.remoter.run("sudo bash -cxe '%s'" % clean_script)
+            output = self.remoter.run('sudo grep replace_address: /etc/scylla/scylla.yaml', ignore_status=True)
+            if 'replace_address:' not in output.stdout:
```

Comment:
this won't work if we will run `restart()` method more than one time and the on each of the restarts we get 
different IP

Reply:
This is the internal IP address and it should remain the same IP.